### PR TITLE
Switch the default renderer to WebRender

### PR DESCRIPTION
**Pull request by glennw**
@larsbergstrom I updated your PR to switch the WPT test runner to use WR, make it default on the command line, and include the differing test expectations.

I'd personally prefer we had time to fix / update some of those test expectations properly - but if you feel it's urgent enough to merge for the tech demo, we could file follow up bugs to fix up some of the differing tests / expectations.

Thoughts?

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/11923)

### CODE REVIEW

File: components/util/opts.rs [hunk 1]
```diff
@@ -742,9 +742,7 @@ pub fn from_cmdline_args(args: &[String]) -> ArgumentParsingResult {
         opt_match.opt_present("b") ||
         !prefs::get_pref("shell.native-titlebar.enabled").as_boolean().unwrap();
 
-    let use_webrender =
-        (prefs::get_pref("gfx.webrender.enabled").as_boolean().unwrap() || opt_match.opt_present("w")) &&
-        !opt_match.opt_present("z");
+    let use_webrender = !opt_match.opt_present("c");
```
Comment (Ms2ger) [inline components/util/opts.rs]: Should we complain if someone passes `-c -w`?

**Comment by larsbergstrom** [comment]
@glennw Thanks for doing this! I think that @Ms2ger and I are in agreement that this is great to land and we'll just plan to follow up on the tests. We certainly weren't likely to get those tests fixed when they weren't being run at all in CI!

**Comment by jdm** [comment]
Lots of errors like:
```
▶ CRASH [expected FAIL] /2dcontext/building-paths/canvas_complexshapes_arcto_001.htm
  │ 
  │ Xlib:  extension "XFree86-VidModeExtension" missing on display ":0".
  │ Failed to compile shader: 0:1(10): error: GLSL 1.50 is not supported. Supported versions are: 1.10, 1.20, 1.30, and 1.00 ES
  │ 
  │ thread 'main' panicked at '-- Shader compile failed - exiting --', /home/servo/.cargo/git/checkouts/webrender-c3596abe1cf4f320/master/src/device.rs:865
  │ stack backtrace:
  │    1:     0x7f4528e7a7df - std::sys::backtrace::tracing::imp::write::h6528da8103c51ab9
  │    2:     0x7f4528e8240b - std::panicking::default_hook::_$u7b$$u7b$closure$u7d$$u7d$::hbe741a5cc3c49508
  │    3:     0x7f4528e8206f - std::panicking::default_hook::he0146e6a74621cb4
  │    4:     0x7f4528d609b7 - util::panicking::initiate_panic_hook::_$u7b$$u7b$closure$u7d$$u7d$::_$u7b$$u7b$closure$u7d$$u7d$::he2b22674ad1748f3
  │    5:     0x7f4528e6806c - std::panicking::rust_panic_with_hook::h983af77c1a2e581b
  │    6:     0x7f4527613c1f - std::panicking::begin_panic::h0bf39f6d43ab9349
  │    7:     0x7f4527622ce3 - webrender::device::Device::compile_shader::h84493a9aecd11731
  │    8:     0x7f45276240c1 - webrender::device::Device::load_program::h0b067737c8558155
  │    9:     0x7f452761d4a0 - webrender::device::Device::create_program::h450a6de873b793fd
  │   10:     0x7f45276c78be - webrender::renderer::Renderer::new::h0b1c23609e0eec8d
  │   11:     0x7f452700903f - servo::main::h6924e5658370f6b5
  │   12:     0x7f4528e819e8 - std::panicking::try::call::h852b0d5f2eec25e4
  │   13:     0x7f4528e8c84b - __rust_try
  │   14:     0x7f4528e8c7ee - __rust_maybe_catch_panic
  │   15:     0x7f4528e8148e - std::rt::lang_start::hfe4efe1fc39e4a30
  │   16:     0x7f45244fef44 - __libc_start_main
  │   17:     0x7f4527007d79 - <unknown>
  └   18:                0x0 - <unknown>
```

**Comment by glennw** [comment]
Rebased this over conflicts, and to get the WR update which lowers shader version on Linux.

**Comment by KiChjang** [comment]
That's... 2103 test failures and 4 test passes.

**Comment by glennw** [comment]
Superseded by #13382.

**Pull request closed without merging**